### PR TITLE
Update shortcodes.csv

### PR DESCRIPTION
**Pull request by SM-102Moderna**
0117 has been added (Rome in the Early Middle Ages: Arts and Culture)

### CODE REVIEW

**Comment by flavens** [comment]
Caution: the project Rome has the shortcode nr. 0118, not 0117!!!!!

**Comment by SM-102Moderna** [comment]
Ok.  fix it. Why the ARK-list was not up to date?

**Comment by flavens** [comment]
I added 2 new Lausanne projects to the ARK registry. But no one tells me to do anything else.

**Comment by SM-102Moderna** [comment]
I thought after adding new projects, the ARK list needs a pr. All I know Daniela told me.

**Comment by SM-102Moderna** [comment]
Then please update the list. Otherwise more errors may occur.

**Comment by flavens** [comment]
Is there another file except for the shorcodes and the ark registry I have to update?

**Comment by SM-102Moderna** [comment]
Welcome to the club ;) I'd like to have such a list too. Other docs: As far as I know, no.

File: data/shortcodes.csv [hunk 1]
```diff
@@ -20,6 +20,7 @@ Lausanne range - 0100 to 07FF,,,
 0114,elites-cio,Les membres du CIO de 1894 à 1972 - prosopographie,
 0115,activites-cs,Etude des activités de la Cinémathèque suisse entre 1951 et 1981,
 0116,medframes,"The Medieval Frame of Monumental Portal Sculpture (Pedestals, Canopies, Niches) : Cultural Transfers and Agency (12th-13th Centuries)",
+0117, Rome in the Early Middle Ages: Arts and Culture,
```
Comment (flavens) [inline data/shortcodes.csv:23]: 0118

**Pull request closed without merging**